gem/oq-engine
# Bin should not be a python package

`bin` in the top project root contained an `__init__.py` file, making the folder a python package.

This is wrong and caused `bin` to be installed in `lib/pythonX.Y/[site|dist]-packages/bin`. Instead `oq` is correctly installed in the right `bin` folder (i.e. `venv/bin`, `/usr/bin`, `/usr/local/bin` or any bin specified with `pip install --install-scripts=`) by `setup.py` (https://github.com/gem/oq-engine/blob/master/setup.py#L119)

How to test:
```
cd oq-engine
virtualenv env
source env/bin/activate
pip install --no-deps .
```
`oq` must exist inside venv's bin
```bash
ls -l env/bin/oq
```
and no bin should exists in `env/lib/python2.7/site-packages`.

Should be backported to `engine-2.3` too.
